.travis.yml

```diff
@@ -1,15 +1,19 @@
 language: go
 sudo: false
-go:
-  - 1.7
-  - 1.8
-  - tip
 go_import_path: go.uber.org/dig
+
+matrix:
+  include:
+    - go: 1.7
+    - go: 1.8
+    - go: tip
+      env: LINT=1
+
 cache:
   directories:
     - vendor
 install:
   - make dependencies
 script:
-  - make lint
+  - test "$LINT" -eq 1 && make lint || echo "Skipping lint"
```

Comment:
Ugh, I can't read.

Ok, lets remember to swap it out for 1.9 when it's released.